Yoast/YoastSEO.js
# Fix precedence exception bug for Spanish and French passive

## Summary

This PR can be summarized in the following changelog entry: 

* Fixes a bug that broke a filter which marks Spanish and French sentences as non-passive when certain exception words occur between the auxiliary and the participle.

## Relevant technical choices:

* Adds `language` to the participle attributes.

## Test instructions

This PR can be tested by following these steps:

In the browserified stand-alone:
* Check out this branch. Dont forget to use `grunt build:js`
* Set the language to `es_ES`
* Paste this sentence: `Los niños que son famosos estaban en un programa de televisión muy esperado.`
* Make sure it is not marked as passive.

NB: This branch was also tested in WordPress SEO. 

Fixes #1429
